Sources/News.php

```diff
@@ -286,16 +286,16 @@ function ShowXmlFeed()
 		ob_start();
 
 	if ($xml_format == 'smf' || isset($_REQUEST['debug']))
```

Comment:
I plan on redoing the whole news system to have each of the output formats as a template because that makes vastly more sense than what has happened here.